content/docs/extensions/timescaledb.md

```diff
@@ -0,0 +1,235 @@
+---
+title: The timescaledb extension
+subtitle: timescaledb is an extension for handling time-series data.
+enableTableOfContents: true
+---
+
+`timescaledb` enables efficient storage and retrieval of time-series data. It is designed to handle large volumes of time-stamped data and provides SQL capabilities on top of a time-oriented data model such as IoT data, sensor readings, financial market data, and other time-series datasets.
+
+<Admonition type="note">
+`timescaledb` is an open-source extension for Postgres that can be installed on any Neon Project using the instructions below.
+</Admonition>
+
+**Version Availability:**
+
+The version of `timescaledb` available on Neon varies based on which version of Postgres you select for your project.
+
+- Postgres 14 - `timescaledb` 2.10.1
+- Postgres 15 - `timescaledb` 2.10.1
+- Postgres 16 - `timescaledb` 2.13.0
+
+_Only Apache-2 licensed features are supported. Compression is not supported._
```

Comment:
Provide a link to the timescaledb editions so that users understand what's included in the Apache-2 version.
https://docs.timescale.com/about/latest/timescaledb-editions/